### PR TITLE
[WIP] [MyX] - faster_rcnn_resnet101_coco-tf accuracy drop

### PR DESCRIPTION
**Pull request by DariaMityagina**
### Details:
Ticket: 65795.
The drop was caused by the changes described in Ticket: 58776.

### CODE REVIEW

File: model-optimizer/extensions/front/tf/pad_tf_to_pad.py [hunk 1]
```diff
@@ -34,6 +34,12 @@ def find_and_replace_pattern(self, graph: Graph):
                 # the input with fill value is an optional third input in TF
                 if not tfpad.in_port(2).disconnected():
                     tfpad.in_port(2).get_connection().set_destination(new_pad.in_port(3))
+                else:
+                    # create Constant node of proper data type (equal to the data type of the Pad first input)
+                    convert_pad_value = create_op_with_const_inputs(graph, ConvertLike, {0: 0.0},
+                                                                    {'name': original_name + '/pad_value_convert'})
+                    convert_pad_value.in_port(1).connect(new_pad.in_port(0).get_source())
+                    new_pad.in_port(3).connect(convert_pad_value.out_port(0))
```
Comment (popovaan) [inline model-optimizer/extensions/front/tf/pad_tf_to_pad.py:42]: @rkazants 
This code was removed in this PR: https://github.com/openvinotoolkit/openvino/pull/6506
For fixing this issue: https://jira.devtools.intel.com/browse/CVS-58776
Is there reason why it was removed?

Comment (popovaan) [reply model-optimizer/extensions/front/tf/pad_tf_to_pad.py:42]: Returning this code back may return the initial issue.

**Comment by DariaMityagina** [comment]
@rkazants could you please take a look?

**Comment by DariaMityagina** [comment]
**Results (full dataset):
Without this fix. MX.**
coco_precision: 19.98%
coco_orig_precision: 19.57%


**With this fix. MX.**
coco_precision: 35.75%
coco_orig_precision: 34.83%

**CPU.**
coco_precision: 35.72%
coco_orig_precision: 34.83%

File: model-optimizer/extensions/front/tf/pad_tf_to_pad.py (hunk 1 above)
Comment (rkazants) [reply model-optimizer/extensions/front/tf/pad_tf_to_pad.py:42]: @DariaMityagina, according to the specification for Pad: "All new elements are populated with 0 if input not provided." So according to the specification this addition is not needed. 
If you have a plugin issue, please create plugin nGraph transformation to explicitly set fill_value.

This is not a defect of MO

**Review by rkazants — CHANGES_REQUESTED** [review]
Not a defect of MO. Please implement plugin specific nGraph transformation, if you plugin needs it.

File: inference-engine/src/transformations/include/transformations/common_optimizations/pad_fillvalue.hpp [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    PadFillValue();
+};
+
+class ngraph::pass::PadFill: public ngraph::pass::GraphRewrite {
+public:
+    NGRAPH_RTTI_DECLARATION;
+    PadFill() {
+        add_matcher<ngraph::pass::PadFillValue>();
+    }
+};
```
Comment (GlebKazantaev) [inline inference-engine/src/transformations/include/transformations/common_optimizations/pad_fillvalue.hpp:36]: You don't need this.

File: inference-engine/src/transformations/include/transformations/common_optimizations/pad_fillvalue.hpp [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+class TRANSFORMATIONS_API PadFillValue;
+
+}  // namespace pass
+}  // namespace ngraph
+
+/**
+ * @ingroup ie_transformation_common_api
+ * @brief 
+ * 
+ */
```
Comment (GlebKazantaev) [inline inference-engine/src/transformations/include/transformations/common_optimizations/pad_fillvalue.hpp:23]: Please add tests for this transformation.

File: inference-engine/src/transformations/include/transformations/common_optimizations/pad_fillvalue.hpp [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+class ngraph::pass::PadFillValue: public ngraph::pass::MatcherPass {
```
Comment (GlebKazantaev) [inline inference-engine/src/transformations/include/transformations/common_optimizations/pad_fillvalue.hpp:24]: Please explain the purpose of this transformation.

**Review by GlebKazantaev — CHANGES_REQUESTED** [review]
It is not clear how this transformation helps to resolve some accuracy issues. There is not tests and no description.

**Comment by DariaMityagina** [comment]
@GlebKazantaev thank you for the comments, but this PR is no longer relevant. Closing it.

**Pull request closed without merging**